src/NHibernate/Dialect/Function/ClassicCountFunction.cs

```diff
@@ -14,9 +14,11 @@ public ClassicCountFunction() : base("count", true)
 		{
 		}
 
+		// Since v5.3
+		[Obsolete("Use GetReturnType method instead.")]
```

Comment:
This PR still preserves `ClassicCountFunction` behavior but creates a risk of losing it.

I think we need to have some non-obsolete override already added in this class for ensuring dropping obsolete members will not result in losing the behavior of this class.

Reply:
Agree, I forgot to add it.